x-pack/legacy/plugins/maps/public/layers/sources/es_search_source/update_source_editor.js

```diff
@@ -226,7 +222,15 @@ export class UpdateSourceEditor extends Component {
           />
         </EuiFormRow>
 
-        {topHitsCheckbox}
+        <EuiFormRow>
+          <EuiSwitch
+            label={i18n.translate('xpack.maps.source.esSearch.useTopHitsLabel', {
+              defaultMessage: `Show most recent documents by entity`,
```

Comment:
change language to something like:

```Show top documents based on sort order```

Reply:
we discussed off line, and can use separate PR to clean up source-settings, as it is getting a real grab bag now (https://github.com/elastic/kibana/issues/47467)

However, in this PR, I would move the ```Show ... by entity``` switch up under the sort definition already. Since right now, the ordering of the options is unrelated. (ie. dynamic filtering switch looks like it doesn't belong).